config.sh

```diff
@@ -21,14 +24,26 @@ export NEKOPATH=$HVM/versions/neko/$NEKO
 export DYLD_FALLBACK_LIBRARY_PATH=$NEKOPATH
 
 # install haxe if needed
+
 if [ ! -d "$HAXEPATH" ]; then
 	mkdir -p "$HVM/versions/haxe"
 
 	ARCHIVE="$HAXEPATH.tar.gz"
-	URL="http://old.haxe.org/file/haxe-$HAXE-osx.tar.gz"
+
+	case $PLATFORM in
```

Comment:
I was thinking that perhaps we should just have another var for platform to avoid these switches, so we can just do:

```shell
URL="http://old.haxe.org/file/haxe-$HAXE-$PLATFORM.tar.gz"
```

Of course we have the annoying inconsistency of mac vs osx. I guess we could just have to vars but that's a bit messy. Cleaner than all these switched though?